tests/python/gaia-ui-tests/gaiatest/apps/ring_tone/app.py

```diff
@@ -9,13 +9,17 @@
 
 class RingTone(Base):
     name = 'Ringtones'
+    origin = 'app://ringtones.gaiamobile.org'
```

Comment:
You shouldn't use this. We're going to implement something in bug 1188926, so that you don't use this, but only need the name